<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bookClassClass</class>
 <widget class="QWidget" name="bookClassClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>869</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>bookClass</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/7b78ceea29ae3cd6dbf27aa34bac4892.jpeg</normaloff>icon/7b78ceea29ae3cd6dbf27aa34bac4892.jpeg</iconset>
  </property>
  <widget class="QPushButton" name="returnBook">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>40</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>归还图书</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-50</x>
     <y>20</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="bookAdd">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>40</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>添加图书</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bookDelete">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>40</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>删除图书</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>1001</width>
     <height>781</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="borrowBook">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>40</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>借阅图书</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newButton">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>40</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icon/bd2ca828946a79181a7b04b264820cc2(1).jpg</normaloff>icon/bd2ca828946a79181a7b04b264820cc2(1).jpg</iconset>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
